Sources/SwiftDocC/Utility/FoundationExtensions/String+Capitalization.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension StringProtocol {
+    
+    /// Returns the string with the first letter capitalized.
+    /// This auto-capitalization only occurs if the first word is all lowercase and contains only characters A-Z.
+    /// The first word can also contain punctuation (e.g. a period, comma, hyphen, semi-colon, colon).
+    func capitalizeFirstWord() -> String {
+        
+        let firstWord = self.prefix(while: { !$0.isWhitespace && !$0.isNewline })
+        
+//        let firstWord = self.components(separatedBy: " ").first ?? ""
+        
+        let firstWordCharacters = CharacterSet.init(charactersIn: String(firstWord))
+        
+        let acceptableCharacters = CharacterSet.lowercaseLetters.union(CharacterSet.punctuationCharacters)
```

Comment:
What's the reasoning behind the rule about "and contains only characters A-Z"? Maybe elaborate a bit more about this here or better in the doc comment above.

Reply:
Fixed the documentation! This comment was added before switching over to the lowercase letters character set, which now provides support for characters in other languages (e.g. ç, é, etc)